--- /etc/ssh/sshd_config        2022-09-22 20:59:45.000000000 +0000
+++ /etc/ssh/sshd_config.new    2022-09-23 08:40:31.608109705 +0000
@@ -19,8 +19,13 @@
 KeyRegenerationInterval 3600
 ServerKeyBits 1024

+Ciphers chacha20-poly1305@openssh.com,aes128-gcm@openssh.com,aes256-gcm@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr
+HostKeyAlgorithms ssh-ed25519,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521
+KexAlgorithms ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521
+MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com
+
 # Logging
-SyslogFacility AUTH
+SyslogFacility AUTHPRIV
 LogLevel INFO

 # Authentication:
@@ -49,7 +54,7 @@
 ChallengeResponseAuthentication no

 # Change to no to disable tunnelled clear text passwords
-#PasswordAuthentication yes
+PasswordAuthentication no

 # Kerberos options
 #KerberosAuthentication no
